Assets/HoloToolkit/UX/Scripts/BoundingBoxes/BoundingBoxRig.cs

```diff
@@ -489,13 +489,16 @@ private List<Vector3> GetBounds()
                 BoundingBox.GetMeshFilterBoundsPoints(clone, bounds, mask);
                 Vector3 centroid = boxInstance.TargetBoundsCenter;
                 GameObject.Destroy(clone);
+#if UNITY_2017_1_OR_NEWER
                 Matrix4x4 m = Matrix4x4.Rotate(objectToBound.transform.rotation);
                 for (int i = 0; i < bounds.Count; ++i)
                 {
                     bounds[i] = m.MultiplyPoint(bounds[i]);
                     bounds[i] += boxInstance.TargetBoundsCenter;
                 }
-
+#else
+#warning "GetBounds() using rotation is not supported on this version of Unity. Recommend updating to 2017.1 or newer."
```

Comment:
Not sure if we should warn people about this, in this way.

Reply:
Fair enough. I can remove those.